CHARTER.md

```diff
@@ -0,0 +1,104 @@
+
+## Governance Bodies
+The MapLibre Organization has three Governance Bodies which support and represent the broader MapLibre Community:
+
+ * Technical Steering Committee
+ * Governing Board
+ * Voting Members
+
+### Technical Steering Committee
+The Technical Steering Committee discusses technical topics that affect multiple MapLibre projects, coordinates ongoing work, and decides if proposed changes with a major impact should be accepted or not.
+
+The committee meets on a monthly basis on a video call. 
+
+The Technical Steering Committee is open to everyone and there are no special requirements if someone wants to join it.
+
+Some projects may have their own steering committees to discuss issues relevant to just that project (e.g., native, gl-js, …).
+
+
+### Governing Board
+The Governing Board helps the Community to define our scope and direction, and is responsible for the organization finances.
+
+The Board is a group of 5 people who represent the MapLibre Organization.
+
+While the Governing Board’s purpose is not to handle technical topics in general, the Board retains the authority over all decisions the MapLibre Projects make, and may overrule the Technical Steering Committee in case of a dispute.
+
+No more than one person on the Governing Board shall be employed or contracted by a single Third-party organization.
+
+The Board shall have at least the following roles:
+
+ * Treasurer: Takes care of financial transactions and accounting.
+ * Secretary: Writes and publishes meeting notes, manages Voting membership.
+
+The Governing Board meets at least 4 times per year, publishes quarterly financial reports, and shares the content of their meetings with the MapLibre Community to the extent the Board deems appropriate.
+
+### Voting Members
+Voting Members are people who elect the Governing Board and who vote on changes to the MapLibre Charter.
+
+A precondition to becoming a Voting Member is that the person contributed in a non-trivial way or that they donated funds to the MapLibre Organization.
+
+If Voting Members are paid by a Third-party organization for their work on the MapLibre Projects, they shall declare publicly who their employer is. At most 10 percent of the Voting Members shall be employed by a single Third-party organization.
+
+Voting Members can nominate other Community members to receive Voting membership in acknowledgement of their contributions to the MapLibre Projects. The Voting Members then vote on granting the nominee a Voting membership. 
+
+Nominations for new Voting Members are open up to two weeks before the election of the Governing Board and the vote on granting Voting Membership to somebody takes place one week before the election of the Governing Board.
+
+### Governing Board Election
+Nomination process:
+
+ * A person can nominate themselves as a candidate for the Governing Board.
+ * Candidates have to register themselves up to 2 weeks before the election.
+
+Voting process:
```

Comment:
@almccon as the resident geo RCV aficionado, is there anything you'd weigh in on here?

Reply:
Closing for now. Feel free to add comments @almccon...